frontend/src/routes/users/me/+page.svelte

```diff
@@ -0,0 +1,168 @@
+<script lang="ts">
+  import Icon from '$components/icons/Icon.svelte';
+  import user from '$stores/user';
+  import wretch from 'wretch';
+
+  const api = wretch('http://localhost:3000/api');
+
+  let birthdate: string;
+  let isSuccess = false;
+
+  const submitBirthdate = async () => {
+    const response = await api
+      .url('/v1/users/me')
+      .headers({ Authorization: `Bearer ${$user.token}` })
+      .post({ birthdate });
+
+    if (response) {
+      isSuccess = true;
```

Comment:
seems like we also need to update birthday in the store? Or is it a part of another story?